packages/paste-core/components/textarea/src/TextArea.tsx

```diff
@@ -26,65 +25,57 @@ export interface TextAreaProps extends React.TextareaHTMLAttributes<HTMLTextArea
   width?: never;
 }
 
-/* eslint-disable emotion/syntax-preference */
-const TextAreaElement = styled(TextareaAutosize)<TextAreaProps>((props) =>
-  css({
-    appearance: 'none',
-    background: 'transparent',
-    border: 'none',
-    borderRadius: 'borderRadius20',
-    boxShadow: 'none',
-    color: 'inherit',
-    display: 'block',
-    fontFamily: 'inherit',
-    fontSize: 'fontSize30',
-    fontWeight: 'fontWeightMedium',
-    lineHeight: 'lineHeight20',
-    maxHeight: 'size30',
-    outline: 'none',
-    paddingBottom: 'space30',
-    paddingLeft: 'space40',
-    paddingRight: 'space40',
-    paddingTop: 'space30',
-    resize: 'vertical',
-    width: '100%',
-
-    '&::placeholder': {
-      color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
-      fontStyle: 'italic',
-    },
-
-    '&:focus::placeholder': {
-      color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
-    },
-
-    '&:disabled': {
-      color: props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
-      cursor: 'not-allowed',
-      // Fixes value color in Safari
-      '-webkit-text-fill-color': props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
-      '-webkit-opacity': '1',
-    },
-  })
-);
-/* eslint-enable */
+const TextAreaElement = React.forwardRef<HTMLTextAreaElement, TextAreaProps>((props, ref) => {
+  return (
+    <Box
+      as={TextareaAutosize}
```

Comment:
Can this be a `Styled(TextareaAutosize)` private component that we compose instead?

Reply:
We need the element `<textarea />` to be a Box so we can apply the custom styles to it from the theme, so I'm not sure if we can do that? I'd still need to make Box as `StyledTextareaAutosize` for it to pick up the styles

Reply:
I guess I'm not understanding why this wouldn't be the same:

```js
const StyledTextareaAutosize = styled(TextareaAutosize)(css({
      appearance="none"
      backgroundColor="transparent"
      borderWidth="borderWidth0"
      borderRadius="borderRadius20"
      boxShadow="none"
      color="inherit"
      display="block"
      fontFamily="inherit"
      fontSize="fontSize30"
      fontWeight="fontWeightMedium"
      lineHeight="lineHeight20"
      maxHeight="size30"
      outline="none"
      paddingBottom="space30"
      paddingLeft="space40"
      paddingRight="space40"
      paddingTop="space30"
      resize="vertical"
      width="100%"
      _placeholder={{
        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
        fontStyle: 'italic',
      }}
      _focus_placeholder={{
        color: props.variant === 'inverse' ? 'colorTextInverseWeak' : 'colorTextWeak',
      }}
      _disabled={{
        color: props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
        cursor: 'not-allowed',
        // Fixes value color in Safari
        '-webkit-text-fill-color': props.variant === 'inverse' ? 'colorTextInverseWeaker' : 'colorTextWeaker',
        '-webkit-opacity': '1',
      }}
}));


export const TextareaElement = React.forwardRef<HTMLTextAreaElement, TextAreaProps>((props, ref) => {
  return (
    <StyledTextareaAutosize
       {...safelySpreadBoxProps(props)}
       async
      rows={3}
      spellCheck
      size={null}
      height={null}
      ref={ref}
   />
)

Reply:
OK, we now use the box utility on the style component instead. Smaller change, no Box as other component